test/platform/github/index.spec.ts

```diff
@@ -844,6 +844,18 @@ describe('platform/github', () => {
             },
           } as any)
       );
+      api.get.mockImplementationOnce(
```

Comment:
Please use `mockResolvedValueOnce`, then you dont need the lamda function

Reply:
I just followed the code that was here before

Reply:
technicall the lambda function has to return a Promise, because api.get is normally an async function. so you can use `mockResolvedValueOnce` to save some code.

https://jestjs.io/docs/en/mock-function-api.html#mockfnmockresolvedvalueoncevalue

Reply:
Yeah, this looks reasonably, but existing code have a lot of `mockImplementationOnce` for same stuff. I don’t think that I should change it all, as well as change only in this case

Reply:
You don't need to change existing test code if not nessessary, but new code should not follow old mistakes.

Reply:
Ok, done